types/styled-theming/styled-theming-tests.tsx

```diff
@@ -68,5 +68,7 @@ const Button = styled.button`
 
 const elementWithoutProp = React.createElement(Button);
 const elementWithCorrectProp = React.createElement(Button, { kind: 'secondary' });
-// $ExpectError
+// TODO(react): Unclear why this doesn't error. Probably picking an overload where excess props are fine.
 const element = React.createElement(Button, { kind: 'wrong variant' });
```

Comment:
I'm not too concerned about this no longer being a type error since most callsites use JSX anyway. Need to figure this out eventually though.

Reply:
FYI, this might get fixed by [this PR](https://github.com/microsoft/TypeScript/pull/57909), see the comment [here](https://github.com/microsoft/TypeScript/pull/57909#issuecomment-2016569072)